tests/resolve-package-path-test.js

```diff
@@ -8,52 +8,77 @@ var Cache = require('../lib/cache');
 var CacheGroup = require('../lib/cache-group');
 var fixturesPath = path.join(__dirname, 'fixtures');
 
+var fs = require('fs');
+var Project = require('fixturify-project');
+
 describe('resolvePackagePath', function() {
+  var project, foo, fooBar, dedupped, linked, linkedBar, unlinked;
+
+  beforeEach(function() {
```

Comment:
@davecombs  / @rwjblue i converted the fixtures to be programmatically generated by `fixturify-project` to tidy things up

Reply:
Great!